src/Sql/dbo/Stored Procedures/User_UpdateKeys.sql

```diff
@@ -15,7 +15,8 @@ BEGIN
         [Key] = @Key,
         [PrivateKey] = @PrivateKey,
         [RevisionDate] = @RevisionDate,
-        [AccountRevisionDate] = @RevisionDate
+        [AccountRevisionDate] = @RevisionDate,
```

Comment:
❓ Why aren't explicit parameters provided here?

Reply:
Yea, this looks like an existing hack that I just went along with. I can try to fix it here with explicit parameters.

Reply:
Done